pages/404.js

```diff
@@ -0,0 +1,24 @@
+import Link from "next/link"
+import { useEffect } from "react"
+import { useRouter } from "next/router";
+
+const NotFound = () => {
+    const router = useRouter();
+
+    useEffect(() => {
+        setTimeout(() => {
+            router.push("/")
+
+        }, 3000)
```

Comment:
I like this experiment from a purely technical point of view. But as a user experience it's not great when a page surprises me by all of a sudden navigating away. So I would recommend not doing this. 😊